core/src/test/resources/test/VericalMultiline.stat

```diff
@@ -27,12 +27,15 @@ def format_![T <: Tree](code: String)(f: A => B)(implicit ev: Parse[T]): String
 
 <<< curried function mixed style 1
 def format_![T <: Tree](
-    code: String
-  )(implicit ev: Parse[T]): String = 1
+    code: String,
+    code2: String
+  )(implicit ev: Parse[T], ex: D): String = 1
```

Comment:
I would have expected there to be a new line before ex. Are you sure this is correct?